repository/scripts/legacy/interaction/Stairs.js

```diff
@@ -1,169 +0,0 @@
-/**
```

Comment:
What happened to stairs & doors? I see the files removed but nothing new to replace them